mypyc/lib-rt/list_ops.c

```diff
@@ -29,6 +29,15 @@ PyObject *CPyList_Build(Py_ssize_t len, ...) {
     return res;
 }
 
+PyObject *CPyList_Copy(PyObject *list) {
+    if(PyList_CheckExact(list)) {
+        return PyList_GetSlice(list, 0, PyList_GET_SIZE(list));
+    }
+    _Py_IDENTIFIER(copy);
+    return _PyObject_CallMethodIdNoArgs(list, &PyId_copy);
```

Comment:
I'm trying to get rid of `_PyObject_CallMethodIdNoArgs` as it's only an internal method. See #18761
It would be better to use `PyObject_CallMethodNoArgs` here instead.

Reply:
Done